### ISSUE TITLE
Enhance Metatags for improved SEO/usability with social media 

### ISSUE THREAD

**Issue by co60ca** [paragonie/airship#46]
- [ ] Check this box if this is a security vulnerability.

### Summary
Certain metatags are used for embedding summaries in certain social media sites and improving the experience (an potentially rankings) on search aggregators.

A few searches brought up these examples:
http://ogp.me/
The Open Graph protocol is an open standard for these types of things and a minimal subset would likely be trivial to enable given the fields of content types.

Airship should consider targeting Facebook, Twitter, G+ with any specific metatags that focus on them, as well as more general ones like in ogp.me (which I believe satisfies Facebook)

**Comment by paragonie-scott** [comment]
Typo, meant to close 45.

**Comment by paragonie-scott** [comment]
We mostly do meta tags already, but I should probably look into Open Graph.

**Comment by paragonie-scott** [comment]
I'll add meta tags to blog posts in 1.2.0, but I'm going to hold off on Open Graph until later, at which point it will be added to custom pages as well.

**Comment by Aranjedeath** [comment]
Yeah, you wanna have meta-description for sure. The first (iirc) 127 chars of that shows up directly in search results. For blog posts, it's easy to make a twitter card as well ( https://dev.twitter.com/cards/types/summary ) or ( https://dev.twitter.com/cards/types/summary-large-image ) if there's a big image at the top. Facebook reads opengraph tags, when you implement them ( https://developers.facebook.com/docs/sharing/webmasters ). 

Google used to also support  ( http://microformats.org/wiki/hcard ) hcard(harticle/etc) and related formats. It was never announced as a feature, but you could use it and they'd parse the formats. I need to check and see if they still do so. I moved to h-card (the second edition of the markup) and most of the neat stuff that gave me broke in search. But that may also have been related to google getting rid of authorship (which was announced two years ago, but the removal was only fully implemented this spring).

**Comment by paragonie-scott** [comment]
It's actually an even number (150 or 160 -- I know some of our company pages accidentally hit the mark dead on).
